.github/workflows/multi-arch-test-build.yml

```diff
@@ -35,9 +35,17 @@ jobs:
       - name: Determine changed packages
         run: |
           # only detect packages with changed Makefiles
-          PACKAGES="$(git diff --diff-filter=d --name-only origin/master \
-            | grep -E 'Makefile$|test.sh$' | grep -Ev '/files/|/src/' \
-            | awk -F/ '{ print $(NF-1) }' | tr '\n' ' ')"
+          PKG_ROOTS=$(find -name Makefile | sed -e 's@./\(.*\)/Makefile@\1/@')
```

Comment:
I think we have to make sure that there is no `/src/` in the path as some src folders contain a Makefile.

Reply:
Let me test that.

Reply:
good catch, added a 'grep -v "src/"', seems to fix that.